impl/internal/did/did.go

```diff
@@ -62,6 +62,19 @@ func (d DHT) Suffix() (string, error) {
 	return "", fmt.Errorf("invalid did:dht prefix: %s", d)
 }
 
+// IdentityKey returns the ed25519 public key for the DHT identifier
```

Comment:
just a thought but you could link to the spec here in the doc comments https://did-dht.com/#identity-key

Reply:
good call out